.golangci.yml

```diff
@@ -10,106 +10,74 @@ linters:
   enable:
     - asciicheck
     - bidichk
-    #    - bodyclose <- needs go 1.18 support
-    # - contextcheck <- needs go 1.18 support
-    #    - cyclop
     - depguard
-    #    - dogsled <- disabled because we would like to participate in the iditerod
-    # - dupl <- really just catches cli stub text
     - durationcheck
     - errcheck
     - errname
-#    - errorlint <- later
-#    - exhaustive <- too picky
-    #    - exhaustivestruct <- disabled because we don't want to use every element of every struct
     - exportloopref
-    #    - forbidigo <- forbids fmt.Print* by default
     - forcetypeassert
-    #    - funlen <- some of our functions are longer than this by necessity
-    #    - gci <- questionable utility
-    #    - gochecknoglobals <- disabled because we want some globals
-    #    - gochecknoinits <- disabled because we want to use init()
-    #    - gocognit <- disabled for same reason as gocyclo
     - goconst
-    # - gocritic <- re enable later to catch capitalized local variables
-    #    - gocyclo <- disabled because it won't work with how we cosmos
-    # - godot
-    #    - godox <- detects TODO/BUG/FIXME and we may wantnthis later but isn't appropriate now
-    #    - goerr113 <- disabled due to lack of comprehension
-    - gofmt
-    # - gofumpt <- disabled https://github.com/osmosis-labs/osmosis/issues/1309
+    - gocritic # <- re enable later to catch capitalized local variables
+    - gofumpt
```

Comment:
Lets leave this deleted, it doesn't have good IDE support.

Reply:
with gofumpt, what I have been doing, is using it to replace both goimports and gofmt

if your ide supports golangci-lint, it will now gofumpt automagically.

Reply:
Looking at the lints from gofumpt, I really don't think this is a win to have as overhead on every PR/commit